assets/js/app.js

```diff
@@ -39,6 +43,18 @@
             _paq.push(['trackEvent', action, origin, target]);
         }
 
+        function trackOrganization(organization, action, dataset, format) {
+            //todo track view when dataset page is rendered
+            customDimensionDataset = {};
+            customDimensionDataset['dimension'+customDimensionActionDatasetId] = organization;
+            customDimensionFormat = {};
+            customDimensionFormat['dimension'+customDimensionActionFormatId] = organization;
```

Comment:
Shouldn't this be `customDimensionFormat['dimension'+customDimensionActionFormatId] = format;` ?

Reply:
no, we push the format on the trackEvent as name:
https://developer.matomo.org/guides/tracking-javascript-guide#custom-dimensions